ios/StatusIm/main.m

```diff
@@ -1,10 +1,8 @@
 /**
  * Copyright (c) Facebook, Inc. and its affiliates.
- * All rights reserved.
  *
- * This source code is licensed under the BSD-style license found in the
- * LICENSE file in the root directory of this source tree. An additional grant
- * of patent rights can be found in the PATENTS file in the same directory.
+ * This source code is licensed under the MIT license found in the
```

Comment:
how are these changes required for status-go dependency?

Reply:
They aren't, I just ran a RN eject to check for differences and decided to leave the new header. I'll remove this from the PR.